target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+        model = "YunCore CPE830";
+        compatible = "yuncore,cpe830", "qca,qca9531";
+
+	chosen {
+		bootargs = "console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd";
```

Comment:
rootfs type should be autodetected (Or is the Bootloader messing with the bootargs?)

Reply:
bootloader IS messing up. hardware provider (Yuncore) tried to ensure nobody can removes their own software